### DISCUSSION TITLE
orangepi r1 plus RK3328

### DISCUSSION THREAD

**Discussion by jxygit** [Ysurac/openmptcprouter#1804] [Q&A]
When can orangepi r1 plus be supported?
😊Thanks for your hard work！

Configuration：
CPU:Rockchip RK3328
GPU:Mali-450MP2 Supports OpenGL ES 1.0/2.0
Power chip:RK805
RAM:1GB DDR4
Ethernet card：1：0M/100M/1000M 以太网（RTL8211E） 2：10M/100M/1000M USB以太网（RTL8153B）
USB:1*USB 2.0 HOST
Low-level peripherals:13pin-UART、I2C、IR pin、Tv-out、AUDIO(NotMIC)、1*GPIO
Debug serial port：UART-TX、UART-RX、GND
button：1*Reset button
Fan interface：1xFan interface（5V）
LED：Power indicator and network port indicator

**Comment by Ysurac** [comment]
Can be supported but for that I need to own it.
As all platforms, I can buy it if there is enough users interested or if someone send me one or donate for that.

**Reply by jxygit** [reply]
Ok, no problem! But I don't have PayPal to use! Do you have Alipay?

**Reply by Ysurac** [reply]
I don't have Alipay and I'm not sure I can open an account in France

**Reply by jxygit** [reply]
Maybe. You can try? Orange PI R1 plus is made in China. Can you buy it?

**Reply by jxygit** [reply]
![Screenshot_2021-05-03-20-45-37-017_com.android.browser.png](https://user-images.githubusercontent.com/83522876/116877465-901b4700-ac50-11eb-9297-4a3e0adcc6dd.png)

This is the official website:

http://www.orangepi.cn/Orange%20Pi%20R1%20Plus/index_cn.html

**Reply by jxygit** [reply]
> I don't have Alipay and I'm not sure I can open an account in France

There is no PayPal in Google play store. You can try alipayhk

**Reply by Mysteo91** [reply]
Orange Pi R1 Plus it is same that NanoPi R2S
